CONTRIBUTING.md

```diff
@@ -117,7 +117,7 @@ We definitely appreciate pull requests that highlight or reproduce a problem, ev
 
             a. Add new redirect url : <ngrok url>/api/v1/auth/oauth
             b. Click save urls
-            c. Add the following scopes : bot, commands, link:read, link:write, chat:write:bot
+            c. Add the following scopes : channels:read, chat:write, commands, groups:read, im:history, im:read, im:write, links:read, links:write
```

Comment:
Is this `im:history` for the requests threading feature? Or is this a scope that the app already had a need for?

Reply:
It was a scope that the app already requires. The scopes we might need for the requests threading feature would include `channels:history` and `groups:history` as well.